docs/source/cli_help/discover.rst

```diff
@@ -13,6 +13,7 @@ Discovers and displays tree of branch dependencies using a heuristic based on re
 If confirmed with a ``y[es]`` or ``e[dit]`` reply, backs up the current definition file (if it exists) as ``$GIT_DIR/machete~``
```

Comment:
I'd update the docs for `anno` as well, to mention how it interacts with qualifiers (IIUC, they aren't treated specially by `anno` at all)